sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -85,6 +85,11 @@ export class ChatClient {
     const authPolicy = createCommunicationTokenCredentialPolicy(this.tokenCredential);
     this.client.pipeline.addPolicy(authPolicy);
 
+    (options as any).signalingClientOptions = {
```

Comment:
We usually don't mutate the options passed into the client as the consumer might reuse them later. Perhaps it would be helpful to declare an interface that includes `signalingClientOptions` and assign a copy of what you need to that, similar to how we handled `InternalPipelineOptions` above?

Reply:
I've removed the use of `any` and wrapped into an internal type definition.